EXILED/Exiled.API/Features/Npc.cs

```diff
@@ -63,6 +69,107 @@ public override Vector3 Position
             }
         }
 
+        /// <summary>
+        /// Gets or sets the player being followed.
+        /// </summary>
+        /// <remarks>The npc must have <see cref="PlayerFollower"/>.</remarks>
+        public Player? Followed
+        {
+            get
+            {
+                if (PlayerFollower == null)
```

Comment:
wouldnt work if it was added in RA

Reply:
Also if you could make it only like 1 method called Spawn() 

(i know i added the create method but i kinda forgot to name it to that lol)

Reply:
What do you reccomend?

Reply:
Because i've no idea getting it every time its not a good idea for performance else =?? but not sure if that would work